lib/vet360/models/transaction.rb

```diff
@@ -26,7 +26,7 @@ def self.build_from(body)
         Vet360::Models::Transaction.new(
           messages: messages || [],
           id: body['tx_audit_id'],
-          status: body['tx_status']
+          status: body['status']
```

Comment:
I believe we actually do want to capture `tx_status` (or `txStatus` in Vet360 speak)

Reply:
This change is taking this body:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/vet360/contact_information/transaction_response.rb#L13

and trying to call `body['tx_status']` on it; however, here is how the body looks in  the `post_email_success.yml` cassette:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr_cassettes/vet360/contact_information/post_email_success.yml#L47

This body only contains `'status'`.

I think yesterday we were looking at the `email_transaction_status` cassette, which has this body:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr_cassettes/vet360/contact_information/email_transaction_status.yml#L47

However, we aren’t hitting that transaction endpoint in a POST or PUT.  We only hit that endpoint when checking the status.

So unless I am still misunderanding things, this does need to be changed to `'status'` here.  

Would you agree?

Reply:
So the field name changes for the different requests? Weird!

Reply:
Sounds good! 

@emersonthis I think it's just the fact that tx_status isn't returned for POST/PUTs, just for when you actually check the status of the transaction.